commands/login.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (topic) {
     topic: topic,
     command: 'login',
     flags: [{name: 'verbose', char: 'v', hasValue: false}],
-    description: 'login to the Heroku Container Registry',
+    description: 'login to Heroku Container Registry',
```

Comment:
Shouldn't it also be "log in"? "login" is a noun.

Reply:
I think so. I believe that's the style that dev center goes with, but we're not as consistent as we probably should be in the CLI

Reply:
Yes, it should be "log in" and "log out"